### PR TITLE
K8s kafka updates

### PR DESCRIPTION
**Pull request by sagiL-oosto**

### CODE REVIEW

**Review by OfirNAnyvision — APPROVED** [review]

File: automation/devops_automation_infra/k8s_plugins/kafka.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def _expose(self):
 
         logging.debug("Waiting for kafka brokers to restart")
         waiter.wait_for_predicate(lambda: self._kafka_brokers_restarted(pods_timestamps) is True, timeout=30)
-        waiter.wait_for_predicate(lambda: self._is_running is True, timeout=60)
+        waiter.wait_for_predicate(lambda: self._is_running is True, timeout=120)
```
Comment (idonaaman) [inline automation/devops_automation_infra/k8s_plugins/kafka.py:74]: why you increased the timeout? what changed?

File: automation/devops_automation_infra/utils/kubectl.py [hunk 2]
```diff
@@ -269,11 +269,12 @@ def edit_statefulset(client, name, namespace='default', new_env_var=None):
     v1_app.replace_namespaced_stateful_set(name, namespace, sts_info)
 
 
-def add_env_to_deployment(client, name, namespace='default', new_env_var=None):
+def add_env_to_deployment(client, name, namespace='default', envs=None):
```
Comment (idonaaman) [inline automation/devops_automation_infra/utils/kubectl.py:272]: looks good!
you checked that the env is appended to the envs in the spec?
what will happen if i will append the same to env in that spec?

File: automation/devops_automation_infra/k8s_plugins/kafka.py (hunk 1 above)
Comment (sagiL-oosto) [reply automation/devops_automation_infra/k8s_plugins/kafka.py:74]: nothing changed but a couple of times in a row my test felt here and after 2 minutes the is running function returned true so I guess 60 sec wont make a difference, if it fails it fails

File: automation/devops_automation_infra/utils/kubectl.py (hunk 2 above)
Comment (sagiL-oosto) [reply automation/devops_automation_infra/utils/kubectl.py:272]: I checked the first one, didn't try to add the same one twice, don't really make sense

File: automation/devops_automation_infra/k8s_plugins/kafka.py (hunk 1 above)
Comment (idonaaman) [reply automation/devops_automation_infra/k8s_plugins/kafka.py:74]: Ok, pay attention please that if you run not on a local machine but on an aws machine, the timeout is sufficient for this case

File: automation/devops_automation_infra/utils/kubectl.py (hunk 2 above)
Comment (idonaaman) [reply automation/devops_automation_infra/utils/kubectl.py:272]: You are correct, but if the machine has not been cleaned or such a situation occurs? Just thinking about an extreme case but it really shouldn't happen

**Review by idonaaman — APPROVED** [review]

**Pull request merged**